### PR TITLE
Revert "Stage 1 of runestones working with buff and combat-trainer."

### PR DESCRIPTION
**Pull request by Dartellum**
Reverts rpherbig/dr-scripts#4695 

@rpherbig  Could you revert https://github.com/rpherbig/dr-scripts/pull/4695? Two reasons: First, without the updated combat-trainer it is not preparing properly, and two, seems some are getting into the else match with crossing-training and I am not sure how (line 117-121 common-arcana.

### CODE REVIEW

File: common-arcana.lic [hunk 1]
```diff
@@ -109,31 +109,22 @@ module DRCA
     end
   end
 
-  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false, runestone_name = '', runestone_tm = false)
+  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false)
     return false unless abbrev
 
     DRC.bput('prep symb', 'You recall the exact details of the', 'But you\'ve already prepared') if symbiosis
 
-    if runestone_name.nil?
```
Comment (rpherbig) [inline common-arcana.lic:117]: I think your problem is this check.

```
def foo(a = '')
	puts a.nil?
end
 
foo(nil)
foo()
```

> true
> false

File: common-arcana.lic [hunk 2]
```diff
@@ -147,7 +138,7 @@ module DRCA
     command = spell['prep'] if spell['prep']
     command = spell['prep_type'] if spell['prep_type']
 
-    return unless prepare?(spell['abbrev'], spell['mana'], spell['symbiosis'], command, tattoo_tm = false, spell['runestone_name'], spell['runestone_tm'])
```
Comment (rpherbig) [inline common-arcana.lic:150]: I also just noticed that you have `tattoo_tm = false`. It's probably safer to just use `false`.

File: common-arcana.lic [hunk 3]
```diff
@@ -503,14 +475,7 @@ module DRCA
     command = data['prep'] if data['prep']
     command = data['prep_type'] if data['prep_type']
 
-    if data['runestone_name']
-      if !prepare_to_cast_runestone?(data, settings)
-        return
-      end
-    end
-
-    return unless prepare?(data['abbrev'], data['mana'], data['symbiosis'], command, tattoo_tm = false, data['runestone_name'], data['runestone_tm'])
```
Comment (rpherbig) [inline common-arcana.lic:512]: I also just noticed that you have `tattoo_tm = false`. The original used `data['tattoo_tm']`. Why the change?

File: common-arcana.lic [hunk 4]
```diff
@@ -676,7 +641,7 @@ module DRCA
     command = data['prep'] if data['prep']
     command = data['prep_type'] if data['prep_type']
 
-    prepare?(data['abbrev'], data['mana'], data['symbiosis'], command, tattoo_tm = false, data['runestone_name'], data['runestone_tm'])
```
Comment (rpherbig) [inline common-arcana.lic:679]: I also just noticed that you have `tattoo_tm = false`. It's probably safer to just use `false`.

Comment (Dartellum) [reply common-arcana.lic:679]: I think I copied it from line 112 because I accidentally deleted it here on one of the versions.

Comment (Dartellum) [reply common-arcana.lic:679]: Also made this `data['tattoo']`.

File: common-arcana.lic (hunk 3 above)
Comment (Dartellum) [reply common-arcana.lic:512]: I think I copied it from line 112 because I accidentally deleted it here on one of the versions.

File: common-arcana.lic (hunk 2 above)
Comment (Dartellum) [reply common-arcana.lic:150]: Also made it `spell['tattoo_tm'].

File: common-arcana.lic (hunk 1 above)
Comment (Dartellum) [reply common-arcana.lic:117]: Make `runestone_name = nil` ?

**Comment by rpherbig** [comment]
I think that will fix the problem you described
